pig/src/main/java/org/jboss/pnc/bacon/pig/impl/nvr/NvrListGenerator.java

```diff
@@ -42,24 +42,20 @@ public static boolean generateNvrList(String repoZipPath, String targetPath) {
         List<KojiBuild> builds = BuildFinderUtils.findBuilds(repoZipPath, true);
         File outputDirectory = new File(FilenameUtils.getPath(targetPath));
         Report nvrReport = new NVRReport(outputDirectory, builds);
+        String basename = nvrReport.getBaseFilename() + ".txt";
 
         try {
             nvrReport.outputText();
         } catch (IOException e) {
-            log.error("Failed to write file {}.txt: {}", nvrReport.getBaseFilename(), e.getMessage(), e);
+            log.error("Failed to write file {} to {}: {}", basename, outputDirectory, e.getMessage(), e);
             return false;
         }
 
-        File srcFile = new File(outputDirectory, nvrReport.getBaseFilename() + ".txt");
-        File destFile = new File(targetPath);
+        File destFile = new File(outputDirectory, basename);
+        boolean exists = destFile.exists();
 
-        try {
```

Comment:
File was already saved to location on line 47, then it tried to move the file on top of itself I think.